ui/src/config.js

```diff
@@ -12,6 +12,7 @@ const defaultConfig = {
   gaTrackingId: '',
   devActivityPanel: true,
   devFastAccessCoverArt: false,
+  enableFavourites: true,
 }
```

Comment:
I'm gonna add the config option in the server, so you can see how it works for when you implement the `enableStarRating` option